src/MainPage.js

```diff
@@ -5,14 +5,7 @@ import FAQ from './components/FAQ'
 import { ReactComponent as VirusRed } from './assets/virus-red2.svg'
```

Comment:
Have you thought about creating an icon file from which you can import your icons into the components?